docs/build-tools/maven.md

```diff
@@ -0,0 +1,61 @@
+---
+id: maven
+title: Maven
+---
+
+Maven is one of the most common build tools in the JVM ecosystem and it also allows for using scala through the [scala-maven-plugin](https://davidb.github.io/scala-maven-plugin/usage.html).
```

Comment:
```suggestion
Maven is one of the most common build tools in the JVM ecosystem and it also allows for using Scala through [scala-maven-plugin](https://davidb.github.io/scala-maven-plugin/usage.html) and [scalor-maven-plugin](https://github.com/random-maven/scalor-maven-plugin).
```

Reply:
I don't think bloop plugin works with scalor, so maybe we shouldn't mention it?

Reply:
I didn't know. Is there a something we can link to for scalor-maven-plugin users? It seems like an actively maintained project with fairly rich features. If so, we can add a new sentence
```
also allows for using scala through the [scala-maven-plugin](https://davidb.github.io/scala-maven-plugin/usage.html).
The [scalor-maven-plugin](https://github.com/random-maven/scalor-maven-plugin) is not supported, see $LINK for more information.
```

Reply:
> I didn't know. Is there a something we can link to for scalor-maven-plugin users?

I don't think we have anything to point to right now - maybe add a comment that we would appreciate a contribution to bloop?

Reply:
Fair enough, we can skip mentioning scalor-maven-plugin in that case and if somebody asks we'll point to Bloop.